.travis.yml

```diff
@@ -3,7 +3,7 @@ rvm:
   - 2.2.3
 env:
   global:
-    - TF_VERSION="0.6.8"
+    - TF_VERSION="0.6.10"
```

Comment:
I can guess why you've done this and it's good, but can you split this out to a separate commit that describes why?